hooks/snyk-code.sh

```diff
@@ -1,6 +1,25 @@
 #!/usr/bin/env bash
+
 set -eu
-SCRIPT_DIR="$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)"
-bash "${SCRIPT_DIR}"/installation/main.sh
 
-snyk code test "$@"
+# Capture exit code of Snyk Test hook
+set +e
+snyk code test
```

Comment:
```suggestion
snyk code test "$@"
```
We need to pass custom parameters defined by user (for example if user pass `-d` for debugging). Same comment for other scripts.